distributed/shuffle/tests/test_no_pandas_import.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import subprocess
+import sys
+
+import_check_code = """
+import sys
```

Comment:
```suggestion
import sys
from .shuffle import ShuffleWorkerExtension
```
otherwise this test doesn't test anything useful...